src/UI/Implementation/Component/Chart/ProgressMeter/FixedSize.php

```diff
@@ -11,4 +11,21 @@
  */
 class FixedSize extends Standard implements C\Chart\ProgressMeter\FixedSize {
 
+	/**
+	 * @param float|int $width
+	 * @return $this
+	 */
+	public function setWidth($width)
+	{
+		$this->width = $width;
+		return $this;
+	}
+
+	/**
+	 * @return float|int
+	 */
+	public function getWidth()
+	{
+		return $this->width;
+	}
```

Comment:
Hi @qdittrich, sry for the long delay after @Amstutz last reply. I guess this part here is a leftover, since we removed the `width`-propery, right? Please remove it.

Still thanks for the cleanup, this looks really good now. And extra kudos for removing more lines than you have added. I'll merge after the leftover is removed.